apps/services/handlers.py

```diff
@@ -277,3 +278,38 @@ async def read_request_details(university_id: int, user_request_id: int, user=De
         "data": response,
         "message": "Got request details"
     }
+
+
+@services_router.post("/{user_request_id}/student-accommodation/{university_id}/",
+                      name="count_student_accommodation",
+                      response_model=JSENDOutSchema[CountHostelAccommodationIn],
+                      tags=["Student dashboard"])
+async def count_student_accommodation(university_id: int, user_request_id: int,  hostel: CountHostelAccommodationIn,
+                                      user=Depends(get_current_user)):
+    """
+    **Count Student Accommodation**
+
+    **Path**:
+    - **university_id**: user university id
+    - **user_request_id**: user request id
+
+    **Input**:
+    - **start_date_accommodation**: start date of user accommodation
+    - **end_date_accommodation**: end(last) date of user accommodation
+    - **bed_place_id**: user bed place id
+
+    **Return**: summa of accommodation, by month price, dates differance and bed place type
+    """
+    hostel_query = select(hostel_accommodation_view).where(hostel_accommodation_view.c.user_request_id == user_request_id,
```

Comment:
It's the wrong implementation. Endpoint receives param hostel_id from USER (UI). Review of user request has still not been created. Endpoint will be used while the admin review user request. He can change hostel, bed place, dates of accommodation. That's why you don't need to make this query and execute it to know the month cost of hostel accomodation